docs/development_guide/records.rst

```diff
@@ -1,10 +1,14 @@
 Records
 =======
 
-To support hardware provenance, the SiliconCompiler supports automated tracking of a number of execution and place of origin related parameters. Tracking is off by default in the SiliconCompiler, but can be turned on with the :keypath:`option,track` parameter. ::
+To support hardware provenance, the SiliconCompiler supports automated tracking of a number of execution and place of origin related parameters.
```

Comment:
"the SiliconCompiler" -> "SiliconCompiler" is more consistent with our other usage (also check the other instance in this paragraph, "...off by default in the SiliconCompiler...")

bit of a nitpick, and I know it was already like this, but might be good to fix while you're updating this page anyways